payments/swish/seamless-view.md

```diff
@@ -18,120 +18,389 @@ sidebar:
-
-  Merchant-->>-SwedbankPay: POST <Create Sales Transaction> (operation=create-sale)
-  activate SwedbankPay
-  SwedbankPay-->>-Merchant: sales resource
-  activate Merchant
-
-  note left of Merchant: POST containing MSISDN
-  Merchant--x-Browser: Tell consumer to open Swish app
-
-  Swish_API->>Swish_App: Ask for payment confirmation
-  activate Swish_App
-  Swish_App-->>-Swish_API: Consumer confirms payment
-  activate Swish_API
-
-  Swish_API-->>-SwedbankPay: Payment status
-  activate SwedbankPay
-  SwedbankPay-->>-Swish_API: Callback response
-  activate Swish_API
-  Swish_API->>-Swish_App: Start redirect
-  activate Swish_App
-
-  Swish_App--x-Browser: Redirect
-  Merchant->>SwedbankPay: GET <Sales transaction>
-  activate SwedbankPay
-  SwedbankPay-->>-Merchant: Payment response
-  activate Merchant
-  Merchant-->>-Browser: Payment Status
+{% include intent.md %}
+
+### Operations
+
+The API requests are displayed in the purchase flow above.
+You can [create a card `payment`][create-payment] with following `operation`
+options:
+
+* [Purchase][purchase] (We use this value in our examples)
+
+### Purchase
+
+A `Purchase` payment is a straightforward way to charge the card of the payer.
+It is followed up by posting a capture, cancellation or reversal transaction.
+
+An example of an abbreviated `POST` request is provided below. Each individual
+Property of the JSON document is described in the following section.
+An example of an expanded `POST` request is available in the
+[other features section][purchase].
+
+{% include risk-indicator.md %}
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/swish/payments HTTP/1.1
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "payment": {
+        "operation": "Purchase",
+        "intent": "Sale",
+        "currency": "SEK",
+        "prices": [{
+                "type": "Swish",
+                "amount": 1500,
+                "vatAmount": 0
+            }
+        ],
+        "description": "Test Purchase",
+        "generatePaymentToken": false,
+        "generateRecurrenceToken": false,
+        "userAgent": "Mozilla/5.0...",
+        "language": "sv-SE",
+        "urls": {
+            "hostUrls": ["http://example.com"],
+            "completeUrl": "http://example.com/payment-completed",
+            "cancelUrl": "http://example.com/payment-canceled",
+            "paymentUrl": "http://example.com/perform-payment",
+            "callbackUrl": "http://example.com/payment-callback",
+            "logoUrl": "http://example.com/payment-logo.png",
+            "termsOfServiceUrl": "http://example.com/payment-terms.pdf",
+        },
+        "payeeInfo": {
+            "payeeId": "12345678-1234-1234-1234-123456789012",
+            "payeeReference": "CD1234",
+            "payeeName": "Merchant1",
+            "productCategory": "A123",
+            "orderReference": "or123",
+        },
+        "prefillInfo": {
+          "msisdn": "{{msisdn}}"
```

Comment:
Where is the `msisdn` variable defined?

Reply:
Forgot to change the name of the property in the table